README.md

```diff
@@ -188,9 +188,26 @@ Account in the Seed Project, grant the necessary roles to the Seed Service
 Account, and enable the necessary API's in the Seed Project.  Run it as follows:
 
 ```sh
-./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME>
+./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME> <BILLING_ACCOUNT>
 ```
 
+In order to execute this script you must have an account with following list of
+bare-minimum permissions:
+
+- `resourcemanager.organizations.list`
```

Comment:
Are we missing `resourcemanager.projects.list` and `billing.accounts.list`?

Reply:
Yes!
Fixed that in [5e878a1](https://github.com/terraform-google-modules/terraform-google-project-factory/pull/230/commits/5e878a134a8d42dc5405439541013515fa372562)
Thank you for great review, @aaron-lane! Good eye!